04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -339,23 +741,25 @@
     }
    ],
    "source": [
-    "print(results)"
+    "result = qc.results(jobId)\n",
+    "print (result)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### Using profiles\n",
+    "<a class=\"anchor\" id=\"profiles\"></a>\n",
+    "# Using profiles\n",
     "\n",
-    "Different datasets (or versions of the same dataset) may reside on different backend servers and a user may want to work explicitly with a (typically older) dataset. In some cases these servers will be used only by developers or those with restricted access.\n",
+    "Different datasets (or versions of the same dataset) may reside on different backend servers and a user may want to work explicitly with a (typically older) dataset. In some cases these servers will be used only by developers or those with restricted access. External TAP services can be accessed by using profiles.\n",
```

Comment:
When you run `qc.profiles()` right now, no internal profiles are shown (and this is the default case on most days). To make it clearer, and also to combine it with the external profiles sentence, I suggest:

"Sometimes, different datasets ... may reside ... Also, external TAP services are accessible from within Data Lab. In both use cases, the `qc.profiles()` function comes in handy."